packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -157,14 +176,15 @@ export async function generateOutputDirectory(
   if (outputBundleOptions.needsServerGenerated) {
     await generateServer(outputBundleOptions);
   }
+  if (!angularVersion) throw new Error("Could not find the angular version of the application");
```

Comment:
this seems like a guard clause to me which typically happens at the top of methods: go/error-handling-tactics-control-flow#guard-clause

this way we can short circuit all of the above logic since we know this function should fail/throw an error. Also I believe we need braces for all `if` statements: go/tsjs-style#control-flow-statements-blocks